packages/cdk/lib/stacks/engines/cromwell-engine-construct.ts

```diff
@@ -112,8 +119,8 @@ export class CromwellEngineConstruct extends EngineConstruct {
     });
 
     const container = definition.addContainer(serviceContainer.serviceName, {
-      cpu: serviceContainer.cpu,
-      memoryLimitMiB: serviceContainer.memoryLimitMiB,
+      cpu: (params.vCpus ? params.vCpus * 1024 : undefined) || serviceContainer.cpu,
```

Comment:
ditto. By the way, may be we should consider avoiding this duplicated code block?